server/internals/service/service.go

```diff
@@ -224,6 +225,7 @@ func (u *Service) AddTestClient() (*models.Client, error) {
 		Secret:      "absolutelynotasecret!",
 		Name:        "Ex-C",
 		RedirectURI: "http://localhost:5173/oauth2/callback",
+		BackendURI:  os.Getenv("BACKEND_URL"),
```

Comment:
Lets make this variable more clear by renaming it to `TEST_CLIENT_BACKEND_URL` instead of just `BACKEND_URL` which is ambiguous

Reply:
I also agree here.